test_util/Cargo.toml

```diff
@@ -1,23 +0,0 @@
-[package]
```

Comment:
Should the files under `src/` in this directory also be removed? I think the code in those files got moved out to various `test_util` mods.

Reply:
The other files were `git mv`ed into `janus_core/src/test_util`. In my checked out copy of branch `timg/test-util-features`, directory `$(SRCROOT)/test_util` no longer exists.